release/mk-release-snapshots.sh

```diff
@@ -9,23 +9,48 @@
 SQUID_RELEASE=`ls -1 $SQUID_VCS_PATH | cut -d- -f2 | sort -h | tail -n 1`
 
 # Hostname of the CI server where snapshots are built
-SQUID_BUILD_SERVER=buildmaster.squid-cache.org
+SQUID_BUILD_SERVER=build.squid-cache.org
 
 # file Jenkins uses to assemble the list of snapshot files
 outfile=HEAD.out
 
+# arguments:
+# jenkins job name
+# destination directory
+get_artifacts() {
+	local job=$1
+	local outdir=$2
+	local artifactsUrl="https://$SQUID_BUILD_SERVER/job/${job}/lastSuccessfulBuild/artifact/artifacts/*zip*/artifacts.zip"
+	rm artifacts.zip 2>&1 >/dev/null
+	if ! wget --quiet --ca-cert=/etc/ssl/certs/ISRG_Root_X1.pem "$artifactsUrl" ; then
+		echo "could not download artifacts from $artifactsUrl"
+		return 1
+	fi
+ 	rmdir -f artifacts 2>&1 >/dev/null
+	if ! unzip -qq artifacts.zip ; then
```

Comment:
This command may get stuck asking questions if some of the files in the archive already exist on disk. If that is not intentional, then more `unzip` options are needed to prevent those questions from being asked.

```sh
$ unzip -qq t.zip
replace a/f2? [y]es, [n]o, [A]ll, [N]one, [r]ename:
```